lib/av_client.ts

```diff
@@ -346,7 +347,7 @@ export class AVClient implements IAVClient {
    * ```
    * @throws {@link NetworkError | NetworkError } if any request failed to get a response
    */
-    public async castBallot(_affidavit?: Affidavit): Promise<string> {
+    public async castBallot(_affidavit?: Affidavit): Promise<BallotBoxReceipt> {
```

Comment:
I suppose this should return a `promise<string>` ? My notes says this should be the ballot tracking code. I think the BallotBoxReceipt type is deprecated.

Reply:
In this particular case I think it should return an actual receipt. The ballot tracking code is something we use to track ballotCryptograms before casting them, unless i'm misunderstanding something. 

That being said, I dont think we have landed on exactly how a receipt should look like.